CONTRIBUTING.md

```diff
@@ -2,6 +2,46 @@
 
 When contributing to this repository, please first discuss the change you wish to make via issue, or any other method with the owners of this repository before making a change.
 
+## Environment setup
+
+Being able to run tests requires an account or service principal with permission (typically Owner) in your Azure subscription.
+
+1. Prepare the Azure configuration file at `tests/integration/cloud-config-azure.ini`, a template of which is available in [the Ansible repo](https://github.com/ansible/ansible/blob/23a84902cb9599fe958a86e7a95520837964726a/test/lib/ansible_test/config/cloud-config-azure.ini.template). Populate your appropriate credential and resource group information.
```

Comment:
```suggestion
1. Prepare the Azure configuration file at `tests/integration/cloud-config-azure.ini`, a template of which is available in [the Ansible repo](https://raw.githubusercontent.com/ansible/ansible/devel/test/lib/ansible_test/config/cloud-config-azure.ini.template). Populate your appropriate credential and resource group information.
```

Would it potentially be better to link to the main branch's copy of the file instead of a link to a static commit?

Reply:
I prefer permalinks in case the file is ever moved or updated.